ui/pages/settings/networks-tab/networks-tab.test.js

```diff
@@ -1,23 +1,14 @@
 import React from 'react';
 import configureMockStore from 'redux-mock-store';
 import { renderWithProvider } from '../../../../test/jest/rendering';
+import { mockNetworkState } from '../../../../test/stub/networks';
+import { CHAIN_IDS } from '../../../../shared/constants/network';
 import NetworksTab from '.';
 
 const mockState = {
   metamask: {
-    providerConfig: {
-      chainId: '0x539',
-      nickname: '',
-      rpcPrefs: {},
-      rpcUrl: 'http://localhost:8545',
-      ticker: 'ETH',
-      type: 'localhost',
-    },
-    networkConfigurations: {},
-    orderedNetworkList: {
-      chainId: '0x539',
-      rpcUrl: 'http://localhost:8545',
-    },
+    ...mockNetworkState({ chainId: CHAIN_IDS.LOCALHOST }),
+    orderedNetworkList: {},
```

Comment:
Same as the above comment, do you think `orderedNetworkList` should still be populated?

Reply:
ah yeah, i put it back to the way it was before in https://github.com/MetaMask/metamask-extension/pull/26150/commits/059ee02788cac2848847bb12f0dbb5094c578563